org.dash.dash-core.json

```diff
@@ -0,0 +1,87 @@
+{
+    "id": "org.dash.dash-core",
+    "runtime": "org.freedesktop.Platform",
+    "sdk": "org.freedesktop.Sdk",
+    "runtime-version": "24.08",
+    "command": "dash-entrypoint.sh",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--filesystem=~/.dashcore:create",
```

Comment:
use `persist=.dashcore`

Reply:
I'm a bit concerned that with this users will have issues finding their wallets or other information

Assuming a user was previously using Dash Core (so had a ~/.dashcore), and install the flatpak. Flatpak will copy ~/.dashcore into ~/.var/app/org.dash.dash-core/data it appears?

But if a user wants to go find their wallets, they will probably look in ~/.dashcore and then get confused when it is not there, (especially since the directory there still exists). This leads me to simply wanting to keep it as `"--filesystem=~/.dashcore:create",`

Please let me know if I'm missing something?

Reply:
Why would you/the user want to inspect `~/.dashcore`?

Reply:
Wallets are stored in ~/.dashcore (by default). Maybe a user has it setup to auto backup their wallets from ~/.dashcore. If they upgrade to the flatpak, I wouldn't want that to silently break (ie no new backups would be made because flatpak will be writing new wallets into ~/.app/org.dash.dash-core)

Reply:
If that's an important thing, then use it. Otherwise persist is generally better for sandboxing.